<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZcloudDeskUpdaterWidget</class>
 <widget class="QMainWindow" name="ZcloudDeskUpdaterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>296</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>296</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>296</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ZcloudDeskUpdater</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>296</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>296</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QWidget" name="FilletWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>484</width>
        <height>280</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>484</width>
        <height>280</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:4px;
</string>
      </property>
      <widget class="QWidget" name="TopWidget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>484</width>
         <height>160</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>484</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>484</width>
         <height>160</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #02A4FD, stop:1 #1F8BED);border-bottom-left-radius:0px;border-bottom-right-radius:0px;</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>121</x>
          <y>50</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/ZcloudDeskUpdater/Resources/tip.png);
background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>201</x>
          <y>71</y>
          <width>163</width>
          <height>18</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>163</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>163</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>检测到云财税新版本</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>167</x>
         <y>200</y>
         <width>120</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: rgb(102, 102, 102);</string>
       </property>
       <property name="text">
        <string>正在下载最新版本</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPercent">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>200</y>
         <width>40</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: rgb(102, 102, 102);</string>
       </property>
       <property name="text">
        <string>0%</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>228</y>
         <width>424</width>
         <height>10</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>424</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>424</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {border:0px;border-radius:5px; background-color:#ebebeb;}
QProgressBar::chunk{border:0px;border-radius:5px;background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #02A4FD, stop:1 #1F8BED);}
QProgressBar {border:0px;border-radius:5px;text-align: center;}</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZcloudDeskUpdater.qrc"/>
 </resources>
 <connections/>
</ui>
